homework/style.css

```diff
@@ -1,3 +1,62 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+.li-root {
+  border: 1px black solid;
+  box-shadow: 3px 4px grey;
+  margin: 5px;
+}
+.ul-root {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+table {
+  text-align: left;
+}
+
+table tr th {
+  font-weight: bold;
+  padding-right: 20px;
```

Comment:
On a mobile device long repo descriptions look better if you align them at top. Try adding this here:

```css
vertical-align: top;
```